client/src/components/AccountInfo/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class AccountInfo extends Component {
+	state = {};
+	render() {
+		return <h6>accounts links </h6>;
+	}
+}
```

Comment:
I think typically something this minimal would be a SFC instead of a class component, but mostly a cosmetic thing

Reply:
Right I will change it